lib/time_difference.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:timeago/timeago.dart' as timeago;
+
+class TimeDifference {
+  static String postAddedTimeDifference(int timeStamp) {
+    DateTime now = DateTime.now();
+    var date1 = new DateTime.fromMillisecondsSinceEpoch(timeStamp * 1000);
+    final diff = now.difference(date1).inMilliseconds;
+    final fifteenAgo =
```

Comment:
What is mean by firteenAgo ?? Don’t copy paste code as it is .. change variable name according to meaning